app/assets/stylesheets/layout/_booklist.scss

```diff
@@ -0,0 +1,76 @@
+.booklist {
+  $base-border-radius: 3px !default;
+  $base-spacing: 1.5em !default;
+  $action-color: #477DCA !default;
+  $dark-gray: #333 !default;
+  $base-font-color: $dark-gray !default;
+  $booklist-gutter: 1.4em;
+  $booklist-image-width: 150px;
+  $booklist-color: $base-font-color;
+  $booklist-image-vert-alignment: top;
+
+  border-bottom: 1px solid transparentize($booklist-color, 0.9);
+  display: table;
+  margin-bottom: $base-spacing;
+  padding-bottom: 1em;
+  width: 100%;
+
+  .booklist-image,
+  .booklist-content {
+    display: table-cell;
+    vertical-align: $booklist-image-vert-alignment;
+  }
+
+  .booklist-image {
+    padding-right: $booklist-gutter;
+
+    > img {
+      border-radius: $base-border-radius;
+      display: block;
+      height: auto;
+      max-width: none;
+      width: $booklist-image-width;
+    }
+
+    a{
+      height: auto;
+      max-width: none;
+      width: $booklist-image-width;
+      display: block;
+    }
+
+    .booklist-reverse-order & {
+      padding-right: 0;
+      padding-left: 10px;
+    }
+  }
+
+  .booklist-content {
+    width: 100%;
+
+    a{
+      font-size: 1em;
+      color: $bookshelf-red-2;
+    }
+
+    a:hover{
+      text-decoration: underline;
+    }
+
+    h1 {
+      font-size: 1.5em;
+      margin: 0 0 0.5em 0;
+      color: $bookshelf-red-2;
+    }
+
+    h3{
```

Comment:
Selector should have depth of applicability no greater than 2, but was 3<br>Opening curly brace `{` should be preceded by one space